hack/verify-docker-build.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# shellcheck source=/dev/null
+source "$(dirname "$0")/utils.sh"
+
+# check if manager docker image builds
+cd_root_path
+
+# DEBUG CODE BEGIN
+service docker start
+# the service can be started but the docker socket not ready, wait for ready
+WAIT_N=0
+MAX_WAIT=5
+while true; do
```

Comment:
This isn't part of the test runner image?

Reply:
ah it was taken from here

https://github.com/kubernetes/test-infra/blob/master/images/bootstrap/runner.sh#L84

Reply:
unless i'm missing something we don't need to do that in our verify script

Reply:
this was a debug commit to see if docker binaries are built into this image we are using. This PR needs https://github.com/kubernetes/test-infra/pull/13366

Reply:
debug code was removed. Resolving this.